frontend/src/components/GClusterMetrics.vue

```diff
@@ -82,9 +118,25 @@ export default {
       return get(this.shootInfo, 'monitoringPassword', '')
     },
     hasAlertmanager () {
+      const ignoreAlerts = get(this.shootItem, 'metadata.annotations["shoot.gardener.cloud/ignore-alerts"]', 'false')
+      if (isTruthyValue(ignoreAlerts)) {
+        return false
+      }
+
       const emailReceivers = get(this.shootItem, 'spec.monitoring.alerting.emailReceivers', [])
       return emailReceivers.length > 0
     },
+    prefix () {
+      return replace(this.shootTechnicalId, /^shoot-{1,2}/, '')
+    },
+  },
+  methods: {
+    oidcDeploymentUrl (target) {
+      return `https://${target}-${this.shootTechnicalId}.${this.seedIngressDomain}`
+    },
+    oidcStatefulsetUrl (target) {
```

Comment:
rename to `getOidcStatefulsetUrl`

Reply:
✅ 
also renamed the other method to `getOidcDeploymentUrl` accordingly